### PR TITLE
Update Helpscout Beacon

### PR DESCRIPTION
**Pull request by OzzieOrca**
(the old beacon is being retired soon)

### CODE REVIEW

File: index.ejs [hunk 1]
```diff
@@ -52,40 +52,32 @@
...
-                        c = e.createElement('script');
-                    (c.type = 'text/javascript'),
-                        (c.async = !0),
-                        (c.src = 'https://djtflbt20bdde.cloudfront.net/'),
-                        r.parentNode.insertBefore(c, r);
-                })(document, window.HSCW || {}, window.HS || {});
-
-                HS.beacon.config({
-                    topArticles: true,
-                });
```
Comment (mattdrees) [inline index.ejs]: This config wasn't part of the old boilerplate code snippet, so it must have been added specifically. I am not sure what it did, but you might double check that something comparable is not needed with the new beacon.

**Comment by mattdrees** [comment]
Can you guys ping me when this is live, and I'll delete the old HS beacon?

File: index.ejs (hunk 1 above)
Comment (OzzieOrca) [reply index.ejs]: I assume that's sorta equivalent to this
Screen Shot 2020-01-16 at 1 51 04 PM
Seems like it's showing mostly the same content as the old one. I don't think it's needed...

**Comment by OzzieOrca** [comment]
@mattdrees I think we are at least waiting on a color change here to `#007398`. Idk what else Spencer might have requested in the email thread.

**Comment by mattdrees** [comment]
I thought I had made that change already, and let Spencer know in slack

**Comment by soberstadt** [comment]
Yeah, I confirmed on the netlify preview the color is right.

**Comment by OzzieOrca** [comment]
Didn't realize the changes were server driven and was waiting for new code or notification that the changes were made. Sorry. Deployed. We can delete the old beacon.

**Comment by mattdrees** [comment]
Great, I deleted it.

**Pull request merged**